app/views/characters/show.html.erb

```diff
@@ -4,6 +4,11 @@
       <div class="text-center">
         <h1><%= @character.name %></h1>
       </div>
+      <% if @character.user %>
+      <h3 class="post-author">
+        <a href="/users/<%= @character.user.id %>">First added by <%= @character.user.username %></a><br>
```

Comment:
This is a nice trick to create expressive/customized links that maintain path flexibility—the following will do the same thing:
```html
<%= link_to @character.user do %>
  First added by <%= @character.user.username %>
<% end %>
```